src/lib/gundb/UserStorageClass.js

```diff
@@ -1194,8 +1213,18 @@ export class UserStorage {
     return status === 'error' ? status : withdrawCode ? otplStatus : ''
   }
 
-  _extractDisplayType(type, withdrawStatus) {
-    return type + `${type === 'send' ? withdrawStatus : ''}`
+  _extractDisplayType(type, withdrawStatus, status) {
+    let sufix = ''
```

Comment:
merge with master, i'm not sure we use this suffix.

Reply:
@sirpy we are using this suffix. You can see how it is used in the previous code version. I just extend the functionality and add the case to modify the display type for the bonus event. The feed event won't be displayed with correct styles without these changes.